docs/development/airflow.md

```diff
@@ -46,6 +46,16 @@ cd src/airflow
 docker build . --tag extending_airflow:latest
 ```
 
+### Set Airflow user ID
+
+!!!note Setting Airflow user ID
+    This command allows Airflow running inside Docker to access the credentials file which was generated earlier.
+
+```bash
+USER_ID=$(id -u)
+sed -Ei "s|^AIRFLOW_UID=.*|AIRFLOW_UID=${USER_ID}|" .env
```

Comment:
Same thing different style. I have no preference:

```bash
echo -e "AIRFLOW_UID=$(id -u)\nAIRFLOW_GID=0" >> .env
```
The risk of this is accidentally running the command twice. Even with that error that the risk is very low, I'd say.

Reply:
`sed` command runs on my environment. I had bad experiences with sed in the past because the mac implementation wasn't the same as the GNU sed

Reply:
The proposed command wasn't working for me because of `-i`. As David says, this works differently. Apparently a good xplatform solution is:
```
USER_ID=$(id -u)
sed "s|^AIRFLOW_UID=.*|AIRFLOW_UID=${USER_ID}|" .env > .env.tmp && mv .env.tmp .env
```

Reply:
@ireneisdoomed @d0choa Based on your comments, and using some of your suggestions, I've designed a new solution which:
* Does not use `sed`
* Does not include an empty variable declaration in the base `.env` file
* Is safe when you run it multiple times
* Is quite simple

I've updated the PR, please do check that it also works on a Mac, and let me know what you think.